README.md

```diff
@@ -0,0 +1 @@
+Scape health inspection data for King County restaurants.
```

Comment:
You could probably find a more descriptive way to phrase this.  What does the scraper use?  How do you run it?  What information does it return?  

Think about what you'd want a visitor to this repository to know about it without them having to read the code.